keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -697,43 +705,58 @@ generatedFrom:
   apiGroup: dra.example.com
   uid: foobar-uid
 
-vendorParameters:
-  # A vendor can put any kind of object here to pass the configuration
-  # parameters down to the kubelet plugin. In this case, the vendor
-  # driver simply copied the entire CR. It could also be some
-  # separate, smaller configuration type.
+# Setup parameters can be provided at two different levels:
+# - for all resources (here)
+# - for a single resource (not shown below)
+#
+# At the moment, only setup parameters defined by a vendor
+# are supported. In-tree definition of common setup parameters
+# might get added in the future.
+setup:
```

Comment:
https://github.com/kubernetes-sigs/wg-device-management/blob/5c4dd792fb42f881ec19c986291131f6c97ce249/k8srm-prototype/pkg/api/claim_types.go#L200 used "config(s)". Let's switch to that.